docs/installation/kubernetes-installation.md

```
@@ -0,0 +1,181 @@
+            model: anthropic/claude-sonnet-4-20250514
+            temperature: 0
+        ```
+
+    === "Azure OpenAI"
+        ```yaml
+        # values.yaml
+        additionalEnvVars:
+        - name: AZURE_API_KEY
+          value: "your-azure-api-key"
+        - name: AZURE_API_BASE
+          value: "https://your-resource.openai.azure.com/"
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)